proto/expr.proto

```diff
@@ -111,13 +111,20 @@ message ExprNode {
     ACOS = 250;
     ATAN = 251;
     ATAN2 = 252;
-    SIND = 253;
-    COSD = 254;
-    COTD = 255;
-    TAND = 256;
-    SQRT = 257;
-    DEGREES = 258;
-    RADIANS = 259;
+    SINH = 253;
+    COSH = 254;
+    TANH = 255;
+    COTH = 256;
+    ASINH = 257;
+    ACOSH = 258;
+    ATANH = 259;
```

Comment:
FYI it would not be backward compatible when we change the mapping of an existing enum value.